src/parser_urdf.cc

```diff
@@ -3466,12 +3466,12 @@ void ReduceSDFExtensionPluginFrameReplace(
         {
           urdf::Vector3 rpy = ParseVector3(rpyKey);
           _reductionTransform.Rot() =
-            ignition::math::Quaterniond::EulerToQuaternion(rpy.x, rpy.y, rpy.z);
+            gz::math::Quaterniond::EulerToQuaternion(rpy.x, rpy.y, rpy.z);
           // remove xyzOffset and rpyOffset
           (*_blobIt)->RemoveChild(rpyKey);
         }
         TiXmlNode* correctedOffsetKey =
-            (*_blobIt)->FirstChild("ignition::corrected_offsets");
+            (*_blobIt)->FirstChild("gz::corrected_offsets");
```

Comment:
these changes to the element name should be reverted

Reply:
[fb6d5f8](https://github.com/gazebosim/sdformat/pull/1118/commits/fb6d5f826605b46774720042e9fa6110baee5090)